.buildkite/pipeline-resource-definitions/kibana-serverless-release.yml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: serverless_release
+  description: Initiate kibana serverless releases
+spec:
+  type: buildkite-pipeline
+  owner: 'group:kibana-operations'
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / serverless / release
```

Comment:
Afaik, the buildkite pipeline ID comes from this field, probably by replacing non-text characters with `-`

Reply:
IIRC it's slugified and yes, it looks like kibana-serverless-release